src/main/scala/org/broadinstitute/dsde/workbench/leonardo/auth/SimpleServiceAccountProvider.scala

```diff
@@ -0,0 +1,36 @@
+package org.broadinstitute.dsde.workbench.leonardo.auth
+
+
+import com.typesafe.config.Config
+import org.broadinstitute.dsde.workbench.leonardo.model.ServiceAccountProvider
+import org.broadinstitute.dsde.workbench.model.WorkbenchEmail
+import org.broadinstitute.dsde.workbench.model.google.GoogleProject
+
+import scala.concurrent.{ExecutionContext, Future}
+
+class SimpleServiceAccountProvider(val config: Config) extends ServiceAccountProvider(config) {
```

Comment:
Just noting this is the exact same implementation as `DefaultServiceAccountProvider`:

https://github.com/DataBiosphere/leonardo/blob/develop/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/auth/DefaultServiceAccountProvider.scala

You could probably just use that one.